### DISCUSSION TITLE
Why is there no json format in the examples?

### DISCUSSION THREAD

**Discussion by jetro4u** [OAI/Arazzo-Specification#217] [Q&A]
I read on the page that 'Arazzo Descriptions can be JSON or YAML values.'
On the documentation page, I saw these two descript
`
`
JSON Templated Example

  contentType: application/json
  payload: |
    {
      "petOrder": {
        "petId": "{$inputs.pet_id}",
        "couponCode": "{$inputs.coupon_code}",
        "quantity": "{$inputs.quantity}",
        "status": "placed",
        "complete": false
      }
    }
JSON Object Example

  contentType: application/json
  payload: 
    petOrder:
      petId: $inputs.pet_id
      couponCode: $inputs.coupon_code
      quantity: $inputs.quantity
      status: placed
      complete: false
`

Which of them apply as a json format for the entire document.

**Comment by handrews** [comment]
@jetro4u I'm not quite sure what you're asking, but the format of the Arazzo Description is independent from the format of the API messages.  If you just want to convert the examples from YAML to JSON there are [online tools that will do that for you](https://www.google.com/search?q=yaml+to+json).

**Reply by jetro4u** [reply]
I was not sure which of the two examples above I should use. I will say I prefer the JSON Templated Example. But I want to know if I can use it to define all the field name in arazzo such as sourceDescriptions, components et al.
